lib/spack/spack/package.py

```diff
@@ -1048,7 +1056,6 @@ def do_fetch(self, mirror_only=False):
                 raise FetchError("Will not fetch %s" %
                                  self.spec.format('{name}{@version}'), ck_msg)
 
-        self.stage.create()
```

Comment:
This is causing the `*_mirror` tests to fail: `URLFetchStrategy` expects `self.stage.path` to exist when it runs `with working_dir(self.stage.path)`.

Reply:
Thanks for catching that!

Reply:
Ouch.  I wonder how/why I removed that line.